### PR TITLE
WFLY-5932 Invalidating a session of an SSO on a different node than w…

### PR DESCRIPTION
**Pull request by stuartwdouglas**
…here the session was created does not logout the user

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
This needs Undertow 1.3.17.Final

File: undertow/src/main/java/org/wildfly/extension/undertow/Host.java [hunk 1]
```diff
@@ -66,6 +66,7 @@
     private final Set<Deployment> deployments = new CopyOnWriteArraySet<>();
     private final Map<String, AuthenticationMechanism> additionalAuthenticationMechanisms = new ConcurrentHashMap<>();
     private final HostRootHandler hostRootHandler = new HostRootHandler();
+    private volatile boolean ssoRegistered;
```
Comment (pferraro) [inline undertow/src/main/java/org/wildfly/extension/undertow/Host.java]: Is this necessary?  Why not just look to see if the SSO authentication mechanism is in the above map?

Comment (stuartwdouglas) [inline undertow/src/main/java/org/wildfly/extension/undertow/Host.java]: Either way works, I don't think it makes that much difference

**Comment by stuartwdouglas** [comment]
Retest this please

**Comment by pferraro** [comment]
This PR is obsolete. See https://github.com/wildfly/wildfly/commit/917fb293dd41afbc2e2a80bf8c41e9309e093284

**Comment by bstansberry** [comment]
Closing per Paul's comment above.

**Pull request closed without merging**